packages/paste-website/src/pages/index.tsx

```diff
@@ -1,10 +1,111 @@
 import * as React from 'react';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
+import {Button} from '@twilio-paste/button';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
 import {SiteWrapper} from '../components/default-site-wrapper';
+import {Grid} from '../components/grid';
+import {StyledHr} from '../components/StyledHr';
+import {Brush} from '../components/icons/Brush';
+import {Terminal} from '../components/icons/Terminal';
+import {WindowLayout} from '../components/icons/WindowLayout';
+
+const LandingContainer = styled(Box)`
```

Comment:
We need a way to use `auto` for `margin-right` and `margin-right`. Once we have that we can get rid of this styled component.

Reply:
It doesn't work? I kind of was working under the assumption it was working

Reply:
@SiTaggart I'm getting the following typescript error when trying to use `auto`:

```
Type '"auto"' is not assignable to type '"space0" | "space10" | "space100" | "space110" | "space120" | "space130" | "space140" | "space150" | "space160" | "space170" | "space180" | "space190" | "space20" | "space200" | "space30" | "space40" | "space50" | "space60" | "space70" | "space80" | "space90" | undefined'.
```

Reply:
ah, typings, right. You can add it to the typings if you wish. Otherwise this is totally fine.

Reply:
I'm going to leave as is for now. I wasn't sure how to add the type to the key without updating the `ThemeShape`

Reply:
Perfectly fine. I created a JIra ticket for it